src/Avalonia.Base/Controls/ResourceDictionary.cs

```diff
@@ -258,6 +255,18 @@ public bool TryGetValue(object key, out object? value)
             return false;
         }
 
+        /// <summary>
+        /// Ensures that the resource dictionary can hold up to <paramref name="capacity"/> entries without
+        /// any further expansion of its backing storage.
+        /// </summary>
+        /// <remarks>This method has no effect when targeting .NET Standard 2.0.</remarks>
+        public void EnsureCapacity(int capacity)
+        {
+#if !NETSTANDARD2_0
```

Comment:
```cs
if(_inner == null)
{
    _inner = new(capacity);
    return;
}
```

Reply:
Good point, the constructor exists on .NET Standard!
I made the change in #15069